src/executables/loader.c

```diff
@@ -6465,3 +6479,36 @@ ldr_act_add_class_all_attrs (LDR_CONTEXT * context, const char *class_name)
 
   return ER_FAILED;
 }
+
+static int
+ldr_json_elem (LDR_CONTEXT * context, const char *str, int len, DB_VALUE * val)
+{
+  JSON_DOC *document = NULL;
+  char *json_body = NULL;
+  int error_code = NO_ERROR;
+
+  error_code = db_json_get_json_from_str (str, document);
+  if (error_code != NO_ERROR)
+    {
+      assert (document == NULL);
+      return error_code;
+    }
+
+  json_body = (char *) db_private_strdup (NULL, str);
+
+  db_make_json (val, json_body, document, false);
```

Comment:
please confirm `false` is correct.

Reply:
it was a mistake, it should've been true